demo.py

```diff
@@ -0,0 +1 @@
+print("practice")
```

Comment:
Please do the following:
1. Add User Interaction (Y/N) - option to proceed or not.
2. Create separate functions for each file type - make modular 
3. Print statements. First give all the files that are to be moved so that user can select whether to proceed. 
4. Print statements : while execute/moving each file